sh_scrapy/crawl.py

```diff
@@ -155,6 +155,27 @@ def _launch():
     _run_usercode(job['spider'], args, _get_apisettings, loghdlr)
 
 
+def list():
```

Comment:
`list` is a builtin name, I know it works but better to name it like it is called: `list_spiders`